### PR TITLE
JanDeDobbeleer.OhMyPosh version 15.5.0

### PR DESCRIPTION
**Pull request by JanDeDobbeleer**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106115)

### CODE REVIEW

**Review by russellbanks — DISMISSED** [review]

**Comment by JanDeDobbeleer** [comment]
@russellbanks I updated the manifest in [this](https://github.com/JanDeDobbeleer/oh-my-posh/commit/4cae83ced1e5530cf17e928aa1cc19e2b03b9397#diff-c1b188aea464233c316d4433e6e6b067668404706ccd12ae2603ad41dcf259aa) commit. I only adjusted the switch as I assumed it would be able to identify the switches based on the scope and inno InstallerType, but I guess that's incorrect.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-user-Locale_Undefined-ErrorScreenshot_20230506222414

File: manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v15.5.0/install-amd64.exe
+  InstallerSha256: AD6125CD93C1A2CBDA3D34A056ADE5AB5E8669C77CCAE0A6D12EF918D92F1378
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml:20]:
```suggestion
    Custom: /INSTALLER=winget /AllUsers
```

File: manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v15.5.0/install-386.exe
+  InstallerSha256: CEA52AA1BA703D1AEF20F81FEE80344918D3415F895D189418868E2BE6570FC8
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml:32]:
```suggestion
    Custom: /INSTALLER=winget /AllUsers
```

File: manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v15.5.0/install-amd64.exe
+  InstallerSha256: AD6125CD93C1A2CBDA3D34A056ADE5AB5E8669C77CCAE0A6D12EF918D92F1378
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v15.5.0/install-amd64.exe
+  InstallerSha256: AD6125CD93C1A2CBDA3D34A056ADE5AB5E8669C77CCAE0A6D12EF918D92F1378
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml:26]:
```suggestion
    Custom: /INSTALLER=winget /CurrentUser
```

File: manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v15.5.0/install-386.exe
+  InstallerSha256: CEA52AA1BA703D1AEF20F81FEE80344918D3415F895D189418868E2BE6570FC8
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v15.5.0/install-386.exe
+  InstallerSha256: CEA52AA1BA703D1AEF20F81FEE80344918D3415F895D189418868E2BE6570FC8
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml:38]:
```suggestion
    Custom: /INSTALLER=winget /CurrentUser
```

File: manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v15.5.0/install-arm64.exe
+  InstallerSha256: 23F0716716C5E950ECADA7DBFF4FC37538DC78C563989CEE1CA48468EAF93493
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
+- Architecture: arm64
+  Scope: user
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v15.5.0/install-arm64.exe
+  InstallerSha256: 23F0716716C5E950ECADA7DBFF4FC37538DC78C563989CEE1CA48468EAF93493
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/15.5.0/JanDeDobbeleer.OhMyPosh.installer.yaml:50]:
```suggestion
    Custom: /INSTALLER=winget /CurrentUser
```

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**